msal-auth-node/app.js

```diff
@@ -0,0 +1,87 @@
+/*
+ This application demonstrates how to include authorization
+ to an API written in Node.Js 16.
+*/
+
+// Node.Js Express Framework
+const express = require('express')
+
+// Used to validate authentication tokens
+const jsonwebtoken = require('jsonwebtoken')
+const jwksRsa = require('jwks-rsa')
+
+// The port the express server will listen on
+const SERVER_PORT = 8080
+
+// 'Directory (tenant) ID' in the Azure portal
+const tenant = ''
+
+const config = {
+  auth: {
+
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    clientId: '',
+
+    // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    authority: `https://login.microsoftonline.com/${tenant}`
+  }
+}
+
+// Scopes required for authentication
+const requiredScopes = ['user_write', 'user_read']
```

Comment:
In the other example that is going in, we used `Message.Read` for the API route.  Since you only have one route, let's try to use the same scope for consistency between then scenarios.

I know you're working on redoing this a bit, but wanted to drop this in here for you to catch for the reworking.

Reply:
Resolving, as this has been updated.